src/fides/common/api/v1/urn_registry.py

```diff
@@ -192,6 +192,8 @@
 SAAS_CONFIG = CONNECTION_BY_KEY + "/saas_config"
 SAAS_CONNECTOR_FROM_TEMPLATE = "/connection/instantiate/{saas_connector_type}"
 REGISTER_CONNECTOR_TEMPLATE = "/connector_template/register"
+CONNECTOR_TEMPLATE_CONFIG = "/connector_template/{key}/config"
+CONNECTOR_TEMPLATE_DATASET = "/connector_template/{key}/dataset"
```

Comment:
I would prefer `connector-template` with a dash, but we're already using underscore for the `register` endpoint